### PR TITLE
fix(ui): few bugs about project and workflow creation

### LINKED ISSUE #4519: fix(ui): when project is created, list is not refreshed

### LINKED ISSUE #4520: fix(ui): disabled button when we want import new workflow via UI

### PR DESCRIPTION
**Pull request by bnjjj**
1. Description
1. Related issues
close #4519 
close #4520 
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-ui#10461.0 ✘  
* Compile
  * Build UI ✔ 
  * TU All -   
  * TU model ✔ 
  * TU shared ✔ 
  * TU store ✘  
  * TU views ✔

**Comment by bnjjj** [comment]
could be interesting to NOT SQUASH in order to revert just one commit if there is a regression

File: ui/src/app/store/project.state.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { IdName, Label, LoadOpts, Project } from 'app/model/project.model';
 import { Usage } from 'app/model/usage.model';
 import { Variable } from 'app/model/variable.model';
 import { NavbarService } from 'app/service/navbar/navbar.service';
+import { ProjectStore } from 'app/service/services.module';
```
Comment (yesnault) [inline ui/src/app/store/project.state.ts]:
```suggestion
import { ProjectStore } from 'app/service/project/project.store';
```

File: ui/src/app/store/project.state.spec.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import { RepositoriesManager } from 'app/model/repositories.model';
 import { Variable } from 'app/model/variable.model';
 import { Workflow } from 'app/model/workflow.model';
 import { NavbarService } from 'app/service/navbar/navbar.service';
+import { ProjectStore } from 'app/service/services.module';
```
Comment (yesnault) [inline ui/src/app/store/project.state.spec.ts]:
```suggestion
import { ProjectStore } from 'app/service/project/project.store';
```

**Comment by ovh-cds** [comment]
CDS Report build-ui#10465.0 ✘  
* Compile
  * Build UI ✔ 
  * TU All -   
  * TU model ✔ 
  * TU shared ✔ 
  * TU store ✘  
  * TU views ✔

**Pull request merged**